upload_form.py

```diff
@@ -107,10 +107,19 @@
             continue
 
 
+# Load the metadata unit mapping
+with open('alternate_allowed_metadata.json') as f:
```

Comment:
@keflavich - Is this json file supposed to be included in the package or is it generated somewhere else?